### PR TITLE
Sony XA2 (H4113)

### PR DESCRIPTION
**Pull request by Ray-Alfy**

### CODE REVIEW

File: pioneer.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        "device": "Xperia XA2",
+        "filename": "crDroidAndroid-10.0-20191206-pioneer-v6.0.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/pioneer/6.x/crDroidAndroid-10.0-20191206-pioneer-v6.0.zip/download",
+        "timestamp": 1575572437,
+        "md5": "5d9870baddc6883f33821b8ec4d0c249",
+        "size": 626122638,
+        "version": "6.0",
+        "buildtype": "Beta",
+        "forum": "https://forum.xda-developers.com/xperia-xa2/development/rom-crdroidandroid-v6-0-t4015327",
+        "forum": "http://4pda.ru/forum/index.php?s=&showtopic=908380&view=findpost&p=91600450",
```
Comment (gwolf2u) [inline pioneer.json]: you can only have one key named forum
please change file and update your pull request

**Review by gwolf2u — CHANGES_REQUESTED** [review]
also you can make a file name changelog_pioneer.txt in format described here 
https://github.com/crdroidandroid/android_vendor_crDroidOTA#12-changelogtxt-structure (changelog structure)

File: pioneer.json (hunk 1 above)
Comment (Ray-Alfy) [reply pioneer.json]: I've no changes bcoz I follow for LOS. And this 1st publish of release

Comment (Ray-Alfy) [reply pioneer.json]: "forum":
did it, only XDA

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**